src/microbit/test/test_image.py

```diff
@@ -63,22 +64,22 @@ def test_blit_heart(self, x, y, w, h, x_dest, y_dest, actual):
         "x, y, w, h, x_dest, y_dest, actual",
         [
             (1, 1, 2, 4, 3, 3, Image("09090:99999:99999:09999:00999:")),
-            (0, 0, 3, 3, 8, 8, Image(CONSTANTS.HEART)),
-            (0, 0, 7, 7, 0, 0, Image(CONSTANTS.HEART)),
+            (0, 0, 3, 3, 8, 8, Image(CONSTANTS.IMAGE_PATTERNS["HEART"])),

```

Comment:
Just curious, why not use `Image.HEART`?

Reply:
Oops, just in the middle of resolving merge conflicts

Reply:
Got it!
Just to be clear since github shows it kind of weird, this is in regards to using `Image.HEART` instead of `Image(CONSTANTS.IMAGE_PATTERNS["HEART"])`.

Reply:
yup! I was explaining why I'm using Image(CONSTANTS.IMAGE_PATTERNS["HEART"])

Reply:
My bad, missed the edit.
Sounds reasonable, thanks for the clarifying!